config/apache/cobbler.conf

```diff
@@ -11,19 +11,19 @@
 
 # This configuration file allows cobbler data to be accessed over HTTP.
 
-AliasMatch ^/cblr(?!/svc/)(.*)?$ "@@webroot@@/cobbler$1"
-AliasMatch ^/cobbler_track(.*)?$ "@@webroot@@/cobbler$1"
+AliasMatch ^/cblr(?!/svc/)(.*)?$ "@@webroot@@/$1"
```

Comment:
Just checking that I understand it correctly, `@@webroot@@` is now set to `/usr/share/cobbler/www/`, right?

Reply:
Exactly; the old webroot was `/srv/www` and that had the `cobbler` dir in it.

Our choice is one of the following:

- Move the webroot to `/usr/share/cobbler/www/` and remove the `cobbler` directory references (since the webroot is already under `cobbler`
- Move the webroot to  `/usr/share/cobbler/www/cobbler` to keep the `cobbler` directory references, but introduce additional `cobbler` directory

I don't mind either; the 2nd option changes fewer things but felt superfluous to me, so I opted for the 1st option. But, if you prefer the 2nd one, we can change that easily :)